### PR TITLE
Bump kube-proxy hotfix version

### PR DESCRIPTION
**Pull request by lodrem**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Review by ShiqianTao — APPROVED** [review]

File: vhdbuilder/packer/generate-windows-vhd-configuration.ps1 [hunk 1]
```diff
@@ -168,14 +168,14 @@ $global:map = @{
         "https://acs-mirror.azureedge.net/kubernetes/v1.21.14/windowszip/v1.21.14-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.22.1-hotfix.20211115/windowszip/v1.22.1-hotfix.20211115-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.22.4-hotfix.20220201/windowszip/v1.22.4-hotfix.20220201-1int.zip",
-        "https://acs-mirror.azureedge.net/kubernetes/v1.22.6-hotfix.20220130/windowszip/v1.22.6-hotfix.20220130-1int.zip",
```
Comment (AbelHu) [inline vhdbuilder/packer/generate-windows-vhd-configuration.ps1:171]: Please do not remove the old one since "v1.22.6-hotfix.20220130" is being used

Comment (AbelHu) [reply vhdbuilder/packer/generate-windows-vhd-configuration.ps1:171]: It is same for others

**Review by AbelHu — CHANGES_REQUESTED** [review]
Please do not remove the old ones since they are being used

File: vhdbuilder/packer/generate-windows-vhd-configuration.ps1 (hunk 1 above)
Comment (lodrem) [reply vhdbuilder/packer/generate-windows-vhd-configuration.ps1:171]: fixed.

**Review by AbelHu — APPROVED** [review]

**Comment by AbelHu** [comment]
@lodrem Thanks for the update. please merge the PR after "AKS Windows VHD Build - PR check-in gate" passes

**Pull request merged**